core/common/lib/policy-engine-lib/src/main/java/org/eclipse/edc/policy/engine/plan/PolicyEvaluationPlanner.java

```diff
@@ -14,11 +14,11 @@
 
 package org.eclipse.edc.policy.engine.plan;
 
-import org.eclipse.edc.policy.engine.spi.AtomicConstraintFunction;
-import org.eclipse.edc.policy.engine.spi.DynamicAtomicConstraintFunction;
+import org.eclipse.edc.policy.engine.spi.AtomicConstraintRuleFunction;
+import org.eclipse.edc.policy.engine.spi.DynamicAtomicConstraintRuleFunction;
 import org.eclipse.edc.policy.engine.spi.PolicyContext;
-import org.eclipse.edc.policy.engine.spi.PolicyValidatorFunction;
-import org.eclipse.edc.policy.engine.spi.RuleFunction;
+import org.eclipse.edc.policy.engine.spi.PolicyValidatorRule;
+import org.eclipse.edc.policy.engine.spi.RulePolicyFunction;
```

Comment:
I guess i missed this in the other PR review (or actually, I missed the other review :/), but I find the name `RulePolicyFunction` a bit ... clunky... How about `PolicyRuleFunction`?

Reply:
I guess it's because "policy function" can be considered an entity on its own, rename makes sense anyway